app/src/main/kotlin/app/lawnchair/lawnicons/ui/component/ClickableIcon.kt

```diff
@@ -37,3 +39,18 @@ fun ClickableIcon(
         )
     }
 }
+
+@Composable
+fun ClickableIcon(
```

Comment:
How about we add an `active` boolean value to highlight the current icon (changing the `tint` value)? This can be useful, especially for large devices, to know what icon you clicked.

Reply:
Is it really that needed? I mean, icon you clicked is displayed in center of the screen🤔